dev/firewalld_integration_tests.sh

```
@@ -0,0 +1,410 @@
+}
+
+# --- Main Execution ---
+main() {
+    info "Starting firewalld integration tests..."
+    
+    info "Waiting for services to be healthy..."
+    retry_count=0
+    max_retries=60  # Longer timeout for firewalld startup
+    retry_interval=2
+    
+    until $(curl --output /dev/null --silent --fail "$BASE_URL/health"); do
```

Shellcheck (0.10.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)